<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Thomas Klammsteiner</title> <meta name="author" content="Thomas Klammsteiner"> <meta name="description" content="microbes | environment | data "> <meta name="keywords" content="microbiome, data science, microbial communities, insect biotechnology, insects, waste management, anaerobic digestion, bioinformatics"> <link rel="stylesheet" href="/al-folio-test/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/al-folio-test/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/al-folio-test/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%9A%9B%EF%B8%8F&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/al-folio-test/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://tklammsteiner.github.io/al-folio-test/"> <link rel="stylesheet" href="/al-folio-test/assets/css/jekyll-pygments-themes-native.css?e74e74bf055e5729d44a7d031a5ca6a5" media="none" id="highlight_theme_dark"> <script src="/al-folio-test/assets/js/theme.js?96d6b3e1c3604aca8b6134c7afdd5db6"></script> <script src="/al-folio-test/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/al-folio-test/">about<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/al-folio-test/blog/">blog</a> </li> <li class="nav-item "> <a class="nav-link" href="/al-folio-test/publications/">publications</a> </li> <li class="nav-item "> <a class="nav-link" href="/al-folio-test/projects/">projects</a> </li> <li class="nav-item "> <a class="nav-link" href="/al-folio-test/teaching/">teaching</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">Thomas</span> Klammsteiner </h1> <p class="desc">microbes | environment | data</p> </header> <article> <div class="profile float-right"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/al-folio-test/assets/img/prof_pic-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/al-folio-test/assets/img/prof_pic-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/al-folio-test/assets/img/prof_pic-1400.webp"></source> <img src="/al-folio-test/assets/img/prof_pic.jpg?4ad390f64e9fd1b08f8da0b3b05a4fcb" class="img-fluid z-depth-1 rounded" width="auto" height="auto" alt="prof_pic.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div class="clearfix"> <p>I’m a microbiologist working on microbial communities and microbe-host interactions. My main work revolves around microbiomes related to <a href="https://www.fromwastetofeed.com/" rel="external nofollow noopener" target="_blank">insect farming</a> and investigating dynamics in insect mass-rearing.</p> <p>Besides that, I am actively contributing to research activities taking place in the Microbial Resource Management group at the <a href="https://www.uibk.ac.at/microbiology/index.html.en" rel="external nofollow noopener" target="_blank">Department of Microbiology</a> and the Molecular Ecology group at the <a href="https://www.uibk.ac.at/ecology" rel="external nofollow noopener" target="_blank">Department of Ecology</a> of the <a href="https://www.uibk.ac.at" rel="external nofollow noopener" target="_blank">University of Innsbruck</a>.</p> <p>My interests include microbiome research, data science and visualization, and making processes more efficient in a sustainable way.</p> </div> <h2><a href="/al-folio-test/news/" style="color: inherit;">news</a></h2> <div class="news"> <p>No news so far...</p> </div> <h2><a href="/al-folio-test/blog/" style="color: inherit;">latest posts</a></h2> <div class="news"> <div class="table-responsive"> <table class="table table-sm table-borderless"> </table> </div> </div> <h2><a href="/al-folio-test/publications/" style="color: inherit;">selected publications</a></h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/al-folio-test/assets/img/publication_preview/frontiers_in_microbiology_2020-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/al-folio-test/assets/img/publication_preview/frontiers_in_microbiology_2020-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/al-folio-test/assets/img/publication_preview/frontiers_in_microbiology_2020-1400.webp"></source> <img src="/al-folio-test/assets/img/publication_preview/frontiers_in_microbiology_2020.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="frontiers_in_microbiology_2020.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="klammsteiner_core_2020" class="col-sm-8"> <div class="title">The core gut microbiome of black soldier fly (<i>Hermetia illucens</i>) larvae raised on low-bioburden diets</div> <div class="author"> <em>Thomas Klammsteiner</em>, Andreas Walter, Tajda Bogataj, Carina D. Heussler, Blaž Stres, Florian M. Steiner, Birgit C. Schlick-Steiner, Wolfgang Arthofer, and Heribert Insam</div> <div class="periodical"> <em>Frontiers in Microbiology</em>, May 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="http://doi.org/10.3389/fmicb.2020.00993" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">doi</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fmicb.2020.00993"></span> </div> <div class="abstract hidden"> <p>An organism’s gut microbiome handles most of the metabolic processes associated with food intake and digestion but can also strongly affect health and behavior. A stable microbial core community in the gut provides general metabolic competences for substrate degradation and is robust against extrinsic disturbances like changing diets or pathogens. Black Soldier Fly larvae (BSFL; <i>Hermetia illucens</i>) are well known for their ability to efficiently degrade a wide spectrum of organic materials. The ingested substrates build up the high fat and protein content in their bodies that make the larvae interesting for the animal feedstuff industry. In this study, we subjected BSFL to three distinct types of diets carrying a low bioburden and assessed the diets’ impact on larval development and on the composition of the bacterial and archaeal gut community. No significant impact on the gut microbiome across treatments pointed us to the presence of a predominant core community backed by a diverse spectrum of low-abundance taxa. Actinomyces spp., Dysgonomonas spp., and Enterococcus spp. as main members of this community provide various functional and metabolic skills that could be crucial for the thriving of BSFL in various environments. This indicates that the type of diet could play a lesser role in guts of BSFL than previously assumed and that instead a stable autochthonous collection of bacteria provides the tools for degrading of a broad range of substrates. Characterizing the interplay between the core gut microbiome and BSFL helps to understand the involved degradation processes and could contribute to further improving large-scale BSFL rearing.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">klammsteiner_core_2020</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{The core gut microbiome of black soldier fly (&lt;i&gt;Hermetia illucens&lt;/i&gt;) larvae raised on low-bioburden diets}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License (CC-BY-NC-ND)}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1664-302X}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/articles/10.3389/fmicb.2020.00993/full?&amp;utm_source=Email_to_authors_&amp;utm_medium=Email&amp;utm_content=T1_11.5e1_author&amp;utm_campaign=Email_publication&amp;field=&amp;journalName=Frontiers_in_Microbiology&amp;id=499002}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fmicb.2020.00993}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{English}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2020-05-21}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Microbiology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Klammsteiner, Thomas and Walter, Andreas and Bogataj, Tajda and Heussler, Carina D. and Stres, Blaž and Steiner, Florian M. and Schlick-Steiner, Birgit C. and Arthofer, Wolfgang and Insam, Heribert}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">may</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Frontiers}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{16S amplicon sequencing, Animal feedstuff, Circular economy, microbial communities, Waste valorization, AGIV, Actinomyces, Larval metabolism}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{993}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/al-folio-test/assets/img/publication_preview/frontiers_in_microbiology_2021C-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/al-folio-test/assets/img/publication_preview/frontiers_in_microbiology_2021C-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/al-folio-test/assets/img/publication_preview/frontiers_in_microbiology_2021C-1400.webp"></source> <img src="/al-folio-test/assets/img/publication_preview/frontiers_in_microbiology_2021C.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="frontiers_in_microbiology_2021C.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="zambrano_applications_2021" class="col-sm-8"> <div class="title">Applications of machine learning in human microbiome studies: a review on feature selection, biomarker identification, disease prediction and treatment</div> <div class="author"> Laura Judith Marcos Zambrano, Kanita Karaduzovic-Hadziabdic, Tatjana Loncar-Turukalo, Piotr Przymus, Vladimir Trajkovik, Oliver Aasmets, Magali Berland, Aleksandra Gruca, Jasminka Hasic Telalovic, Hron Karel, <em>Thomas Klammsteiner</em>, Mikhail Kolev, Leo Lahti, Mart B. Lopes, Victor Moreno, Irina Naskinova, Elin Org, Inês Paciência, Georgios Papoutsoglou, Rajesh Shigdel, Blaz Stres, Baiba Vilne, Malik Yousef, Eftim Zdravevski, Ioannis Tsamardinos, Enrique Carrillo Santa Pau, Marcus Claesson, Isabel Moreno Indias, and Jaak Truu</div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Feb 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="http://doi.org/10.3389/fmicb.2021.634511" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">doi</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fmicb.2021.634511"></span> <span class="__dimensions_badge_embed__" data-doi="10.3389/fmicb.2021.634511" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>The number of microbiome-related studies has notably increased the availability of data on human microbiome composition and function. These studies provide the essential material to deeply explore host-microbiome associations and their relation to the development and progression of various complex diseases. Improved data-analytical tools are needed to exploit all information from these biological datasets, taking into account the peculiarities of microbiome data, i.e. compositional, heterogeneous and sparse nature of these datasets. The possibility of predicting host-phenotypes based on taxonomy-informed feature selection to establish an association between microbiome and predict disease states is beneficial for personalized medicine. In this regard, machine learning (ML) provides new insights into the development of models that can be used to predict outputs, such as classification and prediction in microbiology, infer host phenotypes to predict diseases and use microbial communities to stratify patients by their characterization of state-specific microbial signatures. Here we review the state-of-the-art ML methods and respective software applied in human microbiome studies, performed as part of the COST Action ML4Microbiome activities. This scoping review focuses on the application of ML in microbiome studies related to association and clinical use for diagnostics, prognostics, and therapeutics. Although the data presented here is more related to the bacterial community, many algorithms could be applied in general, regardless of the feature type. This literature and software review covering this broad topic is aligned with the scoping review methodology. The manual identification of data sources has been complemented with: (1) automated publication search through digital libraries of the three major publishers using natural language processing (NLP) Toolkit, and (2) an automated identification of relevant software repositories on GitHub and ranking of the related research papers relying on learning to rank approach.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">zambrano_applications_2021</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Applications of machine learning in human microbiome studies: a review on feature selection, biomarker identification, disease prediction and treatment}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{All rights reserved}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1664-302X}</span><span class="p">,</span>
  <span class="na">shorttitle</span> <span class="p">=</span> <span class="s">{Applications of machine learning in human microbiome studies}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/articles/10.3389/fmicb.2021.634511/abstract}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fmicb.2021.634511}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{English}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2021-02-17}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Microbiology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Zambrano, Laura Judith Marcos and Karaduzovic-Hadziabdic, Kanita and Loncar-Turukalo, Tatjana and Przymus, Piotr and Trajkovik, Vladimir and Aasmets, Oliver and Berland, Magali and Gruca, Aleksandra and Hasic Telalovic, Jasminka and Karel, Hron and Klammsteiner, Thomas and Kolev, Mikhail and Lahti, Leo and Lopes, Mart B. and Moreno, Victor and Naskinova, Irina and Org, Elin and Paciência, Inês and Papoutsoglou, Georgios and Shigdel, Rajesh and Stres, Blaz and Vilne, Baiba and Yousef, Malik and Zdravevski, Eftim and Tsamardinos, Ioannis and de Santa Pau, Enrique Carrillo and Claesson, Marcus and Moreno Indias, Isabel and Truu, Jaak}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">feb</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Frontiers}</span><span class="p">,</span>
  <span class="na">keywords</span> <span class="p">=</span> <span class="s">{microbiome, biomarker identification, Disease Prediction, feature  selection, machine   learning}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{634511}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/al-folio-test/assets/img/publication_preview/waste_and_biomass_valorization_2023-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/al-folio-test/assets/img/publication_preview/waste_and_biomass_valorization_2023-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/al-folio-test/assets/img/publication_preview/waste_and_biomass_valorization_2023-1400.webp"></source> <img src="/al-folio-test/assets/img/publication_preview/waste_and_biomass_valorization_2023.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="waste_and_biomass_valorization_2023.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="heussler_comparative_2023" class="col-sm-8"> <div class="title">A comparative study of effects of biodegradable and non-biodegradable microplastics on the growth and development of black soldier fly larvae (<i>Hermetia illucens</i>)</div> <div class="author"> Carina D. Heussler, Isabel L. Dittmann, Bernhard Egger, Sabine Robra, and <em>Thomas Klammsteiner</em> </div> <div class="periodical"> <em>Waste and Biomass Valorization</em>, Oct 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="http://doi.org/10.1007/s12649-023-02296-0" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">doi</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1007/s12649-023-02296-0"></span> </div> <div class="abstract hidden"> <p>Purpose: This study aimed to investigate the digestion process of biodegradable and non-biodegradable microplastics (MPs) within black soldier fly larvae (BSFL) and assess their impact on larval growth and development. The goal was to understand the fate of MPs within BSFL, considering their potential for waste conversion polluted with MPs. Methods: BSFL were exposed to two types of MPs, and their growth, development, potential accumulation and excretion of MPs were monitored. Results: The findings revealed that the MPs accumulated solely in the larval gut and had no adverse effects on the growth and development of BSFL. Larvae efficiently excreted MPs before reaching the pupation stage. Conclusion: This research emphasizes the potential of BSFL as a bioconversion agent for organic waste, even in the presence of MPs. The effective excretion of MPs by BSFL before pupation suggests their ability to mitigate potential harm caused by MP accumulation. The fact that BSFL may excrete MPs before pupation would contribute to their safe use as animal feedstock. A careful evaluation of the effects of using BSFL reared on contaminated substrates especially containing visually non-detectable residuals like nanoplastics, chemicals or toxic metals and further examination of the broader implications for waste management and sustainable livestock farming remains important.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">heussler_comparative_2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A comparative study of effects of biodegradable and non-biodegradable microplastics on the growth and development of black soldier fly larvae (&lt;i&gt;Hermetia illucens&lt;/i&gt;)}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{All rights reserved}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1877-2641, 1877-265X}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://link.springer.com/10.1007/s12649-023-02296-0}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s12649-023-02296-0}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{en}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-11-06}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Waste and Biomass Valorization}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Heussler, Carina D. and Dittmann, Isabel L. and Egger, Bernhard and Robra, Sabine and Klammsteiner, Thomas}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">oct</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 preview"> <figure> <picture> <source class="responsive-img-srcset" media="(max-width: 480px)" srcset="/al-folio-test/assets/img/publication_preview/frontiers_in_microbiology_2023C-480.webp"></source> <source class="responsive-img-srcset" media="(max-width: 800px)" srcset="/al-folio-test/assets/img/publication_preview/frontiers_in_microbiology_2023C-800.webp"></source> <source class="responsive-img-srcset" media="(max-width: 1400px)" srcset="/al-folio-test/assets/img/publication_preview/frontiers_in_microbiology_2023C-1400.webp"></source> <img src="/al-folio-test/assets/img/publication_preview/frontiers_in_microbiology_2023C.jpg" class="preview z-depth-1 rounded" width="auto" height="auto" alt="frontiers_in_microbiology_2023C.jpg" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="marcos_zambrano_toolbox_2023" class="col-sm-8"> <div class="title">A toolbox of machine learning software to support microbiome analysis</div> <div class="author"> Laura Judith Marcos Zambrano, Víctor Manuel López Molina, Burcu Bakir-Gungor, Marcus Frohme, Kanita Karaduzovic-Hadziabdic, <em>Thomas Klammsteiner</em>, Eliana Ibrahimi, Leo Lahti, Tatjana Loncar-Turukalo, Xhilda Dhamo, Andrea Simeon, Alina Nechyporenko, Gianvito Pio, Piotr Przymus, Alexia Sampri, Vladimir Tihomir Trajkovik, Oliver Aasmets, Ricardo Araujo, Ioannis Anagnostopoulos, Onder Aydemir, Magali Berland, María de la Luz Calle, Michelangelo Ceci, Hatice Duman, Aycan Gundogdu, Aki S. Havulinna, Kardokh Hama Najib Kaka Bra, Eglantina Kalluci, Sercan Karav, Daniel Lode, Marta B. Lopes, Patrick May, Bram Nap, Miroslava Nedyalkova, Inês Paciência, Lejla Pasic, Meritxell Pujolassos, Rajesh Shigdel, Antonio Susin, Ines Thiele, Ciprian-Octavian Truică, Paul Wilmes, Ercüment Yılmaz, Malik Yousef, Marcus Joakim Claesson, Jaak Truu, and Enrique Carrillo De Santa Pau</div> <div class="periodical"> <em>Frontiers in Microbiology</em>, Sep 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> <a href="http://doi.org/10.3389/fmicb.2023.1250806" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">doi</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.3389/fmicb.2023.1250806"></span> </div> <div class="abstract hidden"> <p>The human microbiome has become an area of intense research due to its potential impact on human health. However, the analysis and interpretation of this data have proven to be challenging due to its complexity and high dimensionality. Machine learning (ML) algorithms can process vast amounts of data to uncover informative patterns and relationships within the data, even with limited prior knowledge. Therefore, there has been a rapid growth in the development of software specifically designed for the analysis and interpretation of microbiome data using ML techniques. These software incorporate a wide range of ML algorithms for clustering, classification, regression, or feature selection, to identify microbial patterns and relationships within the data and generate predictive models. This rapid development with a constant need for new developments and integration of new features require efforts into compile, catalog and classify these tools to create infrastructures and services with easy, transparent, and trustable standards. Here we review the state-of-the-art for ML tools applied in human microbiome studies, performed as part of the COST Action ML4Microbiome activities. This scoping review focuses on ML based software and framework resources currently available for the analysis of microbiome data in humans. The aim is to support microbiologists and biomedical scientists to go deeper into specialized resources that integrate ML techniques and facilitate future benchmarking to create standards for the analysis of microbiome data. The software resources are organized based on the type of analysis they were developed for and the ML techniques they implement. A description of each software with examples of usage is provided including comments about pitfalls and lacks in the usage of software based on ML methods in relation to microbiome data that need to be considered by developers and users. This review represents an extensive compilation to date, offering valuable insights and guidance for researchers interested in leveraging ML approaches for microbiome analysis.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">marcos_zambrano_toolbox_2023</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A toolbox of machine learning software to support microbiome analysis}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{14}</span><span class="p">,</span>
  <span class="na">copyright</span> <span class="p">=</span> <span class="s">{All rights reserved}</span><span class="p">,</span>
  <span class="na">issn</span> <span class="p">=</span> <span class="s">{1664-302X}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://www.frontiersin.org/articles/10.3389/fmicb.2023.1250806}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3389/fmicb.2023.1250806}</span><span class="p">,</span>
  <span class="na">language</span> <span class="p">=</span> <span class="s">{English}</span><span class="p">,</span>
  <span class="na">urldate</span> <span class="p">=</span> <span class="s">{2023-11-12}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Frontiers in Microbiology}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Marcos Zambrano, Laura Judith and López Molina, Víctor Manuel and Bakir-Gungor, Burcu and Frohme, Marcus and Karaduzovic-Hadziabdic, Kanita and Klammsteiner, Thomas and Ibrahimi, Eliana and Lahti, Leo and Loncar-Turukalo, Tatjana and Dhamo, Xhilda and Simeon, Andrea and Nechyporenko, Alina and Pio, Gianvito and Przymus, Piotr and Sampri, Alexia and Trajkovik, Vladimir Tihomir and Aasmets, Oliver and Araujo, Ricardo and Anagnostopoulos, Ioannis and Aydemir, Onder and Berland, Magali and Calle, María de la Luz and Ceci, Michelangelo and Duman, Hatice and Gundogdu, Aycan and Havulinna, Aki S. and Kaka Bra, Kardokh Hama Najib and Kalluci, Eglantina and Karav, Sercan and Lode, Daniel and Lopes, Marta B. and May, Patrick and Nap, Bram and Nedyalkova, Miroslava and Paciência, Inês and Pasic, Lejla and Pujolassos, Meritxell and Shigdel, Rajesh and Susin, Antonio and Thiele, Ines and Truică, Ciprian-Octavian and Wilmes, Paul and Yılmaz, Ercüment and Yousef, Malik and Claesson, Marcus Joakim and Truu, Jaak and Carrillo De Santa Pau, Enrique}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">sep</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Frontiers}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1250806}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%74%68%6F%6D%61%73.%6B%6C%61%6D%6D%73%74%65%69%6E%65%72@%75%69%62%6B.%61%63.%61%74" title="email"><i class="fas fa-envelope"></i></a> <a href="https://orcid.org/0000-0003-1280-5159" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid"></i></a> <a href="https://scholar.google.com/citations?user=jI_Q6tgAAAAJ&amp;hl=de" title="Google Scholar" rel="external nofollow noopener" target="_blank"><i class="ai ai-google-scholar"></i></a> <a href="https://www.researchgate.net/profile/Thomas-Klammsteiner/" title="ResearchGate" rel="external nofollow noopener" target="_blank"><i class="ai ai-researchgate"></i></a> <a href="https://github.com/tklammsteiner" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fab fa-github"></i></a> <a href="https://www.linkedin.com/in/tklammsteiner" title="LinkedIn" rel="external nofollow noopener" target="_blank"><i class="fab fa-linkedin"></i></a> <a href="https://twitter.com/fromwastetofeed" title="Twitter" rel="external nofollow noopener" target="_blank"><i class="fab fa-twitter"></i></a> <a href="https://www.fromwastetofeed.com" title="Work" rel="external nofollow noopener" target="_blank"><i class="fas fa-briefcase"></i></a> <a href="https://instagram.com/fromwastetofeed" title="Instagram" rel="external nofollow noopener" target="_blank"><i class="fab fa-instagram"></i></a> </div> <div class="contact-note"> Feel free to reach out via mail. </div> </div> </article> </div> </div> <footer class="sticky-bottom mt-5"> <div class="container"> © Copyright 2023 Thomas Klammsteiner. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/al-folio-test/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/al-folio-test/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/al-folio-test/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/al-folio-test/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/al-folio-test/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/al-folio-test/assets/js/copy_code.js?c9d9dd48933de3831b3ee5ec9c209cac" type="text/javascript"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>